programs/x509/cert_write.c

```diff
@@ -235,6 +235,41 @@ int write_certificate( mbedtls_x509write_cert *crt, const char *output_file,
     return( 0 );
 }
 
+/*
+ * Convert the input "in_buff" string to a raw byte array "out_buff". The amount
```

Comment:
> This program takes the serial number as a base-10 integer, not hex

Actually this is my fault, since I didn't check correctly how the original source code worked. I'll try to adjust it as you suggested ;)